v1.1/create-security-certificates.md

```diff
@@ -1,15 +1,20 @@
 ---
 title: Create Security Certificates
-summary: A secure CockroachDB cluster uses TLS for encrypted inter-node and client-node communication and requires CA, node, and client certificates and keys.
+summary: A secure CockroachDB cluster uses TLS for encrypted inter-node and client-node communication.
 toc: false
 ---
 
-A secure CockroachDB cluster uses [TLS](https://en.wikipedia.org/wiki/Transport_Layer_Security) for encrypted inter-node and client-node communication and requires CA, node, and client certificates and keys. To create these certificates and keys, use the `cockroach cert` [command](cockroach-commands.html) with the appropriate subcommands and flags.
+A secure CockroachDB cluster uses TLS for encrypted inter-node and client-node communication, which requires CA, node, and client certificates and keys. To create these certificates and keys, use the `cockroach cert` [commands](cockroach-commands.html) with the appropriate subcommands and flags, or use [OpenSSL commands](https://wiki.openssl.org/index.php/).
 
-When using `cockroach cert` to create node and client certificates, you will need access to a local copy of the CA certificate and key. It is therefore recommended to create all certificates and keys in one place and then distribute node and client certificates and keys appropriately. For the CA key, be sure to store it somewhere safe and keep a backup; if you lose it, you will not be able to add new nodes or clients to your cluster. For a walkthrough of this process, see [Manual Deployment](manual-deployment.html).
+<div class="filters filters-big clearfix">
+  <button class="filter-button current"><strong>Cockroach Cert Commands</strong></button>
+  <a href="create-security-certificates-openssl.html"><button class="filter-button">OpenSSL Commands</button></a>
+</div>
 
 <div id="toc"></div>
 
+When using `cockroach cert` to create node and client certificates, you will need access to a local copy of the CA certificate and key. It is therefore recommended to create all certificates and keys in one place and then distribute node and client certificates and keys appropriately. For the CA key, be sure to store it somewhere safe and keep a backup; if you lose it, you will not be able to add new nodes or clients to your cluster. For a walkthrough of this process, see [Manual Deployment](manual-deployment.html).
```

Comment:
In the openssl version of the page, you put this info at the start of the `Certificate Directory` section. I like that placement. Can you do the same on this page?

Reply:
Umm..I think in both docs I place the info right after the toc. In any case, I think we need that info in the beginning, because the `Subcommands` section needs it.

Reply:
Hmm. I must've been looking at an outdated version or something. In any case, I'm glad the content is consistently place, but it feels a bit strange orphaned below the TOC. 

- Can you perhaps add a h2 header `Considerations`?
- Can you make sure the formulations are identify on both pages, aside from necessary differences?